pkg/reconciler/v1alpha1/taskrun/resources/input_resource_test.go

```diff
@@ -752,38 +745,6 @@ func TestStorageInputResource(t *testing.T) {
 			},
 		},
 		wantErr: true,
-	}, {
-		desc: "inputs with both resource spec and resource ref",
-		task: &v1alpha1.Task{
```

Comment:
These errors cases are no longer present here since the resource instantiation happens upstream in the calling code